### PR TITLE
active only flag and implementation

### PR DESCRIPTION
**Pull request by boymaas**
Impossible for me to test the implementation as there are hard references to the eoscanada/eos-bios repo. Just posting this as a "probably" working implementation / suggestion. I am not a Go coder, but believe its close enough. So please double check.

### CODE REVIEW

File: bios/mesh.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"math"
 	"math/rand"
 	"time"
+	"sort"
```
Comment (boymaas) [inline bios/mesh.go]: can remove this one. Overlooked

**Comment by abourget** [comment]
You can always develop in the `~/go/src/github.com/eoscanada/eos-bios` directory on your computer :) It won't hurt !

Reading through now :)

**Comment by abourget** [comment]
I see you want to be respectful of what was there.. but do you think we even need `--active-only` ? This will just complicate explainations :)  On launch, we want to mesh with active peers, period.

Can you remove all the options and whatnot.. it could be almost simplified down to `if !peer.Active() { continue }` !

I'd love to merge your PR.. so can you wrap it up?

**Comment by abourget** [comment]
Fixed by 45ffda5672f6a03ee903370b63303232f0e44f69  thanks a lot!!

File: bios/mesh.go [hunk 2]
```diff
@@ -92,10 +94,37 @@ func (b *BIOS) someTopmostPeersAddresses() []string {
...
+	// mesh to those peers deemed Active()
+
+	activeOnly = viper.GetBool("active-only")
 	for _, peer := range b.ShuffledProducers {
-		if peer.Discovery.TargetP2PAddress != "none" {
-			meshableProducers = append(meshableProducers, peer)
-		}
+
+		// skip "none" always
+		if peer.Discovery.TargetP2PAddress != "none" { continue }
```
Comment (oviwankenobi) [inline bios/mesh.go]: mmm?

**Pull request closed without merging**